src/cap32.cpp

```diff
@@ -2079,10 +2084,14 @@ int cap32_main (int argc, char **argv)
       }
       
       while (SDL_PollEvent(&event)) {
+         auto shift_down = static_cast<bool>(event.key.keysym.mod & KMOD_SHIFT);
```

Comment:
This is too early to do this. You have to be sure this is a keyboard event before.
this should be in case SDL_KEYDOWN & case SDL_KEYUP.

Reply:
Ah yes, good point.